tests/contrib/pyramid/test_pyramid.py

```diff
@@ -159,35 +223,17 @@ def json(request):
     config.add_view(json, route_name='json', renderer='json')
     return config.make_wsgi_app()
 
-def includeme(config):
-    pass
-
-def test_include():
-    """ Test that includes do not create conflicts """
-    from ...test_tracer import get_dummy_tracer
-    from ...util import override_global_tracer
-    tracer = get_dummy_tracer()
-    with override_global_tracer(tracer):
-        config = Configurator(settings={'pyramid.includes': 'tests.contrib.pyramid.test_pyramid'})
-        trace_pyramid(config)
-        app = webtest.TestApp(config.make_wsgi_app())
-        app.get('/', status=404)
-        spans = tracer.writer.pop()
-        assert spans
-        eq_(len(spans), 1)
-
-def _get_test_app(service=None):
-    """ return a webtest'able version of our test app. """
-    from tests.test_tracer import get_dummy_tracer
-    tracer = get_dummy_tracer()
-    app = _get_app(service=service, tracer=tracer)
-    return webtest.TestApp(app), tracer
-
 
 if __name__ == '__main__':
     logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
     ddtrace.tracer.debug_logging = True
-    app = _get_app()
+    settings = {
+            'datadog_trace_service': 'foobar',
+            'datadog_tracer': ddtrace.tracer
+            }
```

Comment:
wrong indentation and missing semicolon. Can you manually check flake?
```
flake8 tests/contrib/pyramid
```
I didn't enforce it in the CI, but would be great if we can fix flake in tests too when we're going to rewrite something.